### PR TITLE
review limitations.md

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: apis-overview/limitations.md [hunk 1]
```diff
@@ -2,37 +2,33 @@
...
+In Mindee, you can find these limits in the documentation tab of your API.
+
+![Rate limits part of Mindee's platform documentation tab](https://files.readme.io/56777aa-ratelimits.png)
+
+## Request
+- Maximum request throughput per second: 
+- Maximum request throughput per minute:
+
+## Document
+- Maximum image file size:
```
Comment (fharper) [inline apis-overview/limitations.md:19]: image file = file

we also take PDFs

File: apis-overview/limitations.md [hunk 2]
```diff
@@ -2,37 +2,33 @@
...
-        767,
-        330,
-        "#fcfcfc"
-      ],
-      "caption": "\"Rate limits\" part of Mindee's platform documentation tab."
-    }
-  ]
-}
-[/block]
+[comment]:<what support group should they reach out too>
```
Comment (fharper) [inline apis-overview/limitations.md:8]: I have no idea. I would say the Slack community, but those are decisions for the sales team, so maybe ask in #sales about that.

File: apis-overview/limitations.md [hunk 3]
```diff
@@ -2,37 +2,33 @@
...
 excerpt: ''
 ---
-## Request
 
-A set of limits are enforced to ensure the safety of Mindee's parsing document APIs. Among others:
-* a maximum file size
-* a maximum number of pages for PDF
-* a maximum request throughput per second
-* a maximum request throughput per minute
+A set of limits are enforced to ensure the safety of Mindee's parsing document APIs. A project may have additional limits depending on its plan. An API call that causes any of these limits to be exceeded will be rejected with an error. If you have needs beyond these limits then get in touch with the support team and we'll work something out.
```
Comment (fharper) [inline apis-overview/limitations.md:6]: and we'll work something out

I wouldn't say that as for X reasons, we may not make it work: it's a case by case. I would stop at "team"

File: apis-overview/limitations.md [hunk 4]
```diff
@@ -2,37 +2,33 @@
...
+- Maximum request throughput per second: 
+- Maximum request throughput per minute:
+
+## Document
+- Maximum image file size: 
+- Maximum number of pages for PDF:
+ 
+## API Calls
+- **Free Plan**: Maximum number of API calls: 250 calls
+- **Unlimited Plan**: Maximum number of API calls:
```
Comment (fharper) [inline apis-overview/limitations.md:24]: I think the colon at the end is a typo?

**Review by fharper — APPROVED** [review]
LGTM

**Comment by fakela** [comment]
@fharper I made additional changes by adding to the values, please take a look.

File: apis-overview/limitations.md (hunk 2 above)
Comment (fharper) [reply apis-overview/limitations.md]: I asked them

File: apis-overview/limitations.md (hunk 3 above)
Comment (fharper) [reply apis-overview/limitations.md]: Wait on modifying this one as I think we should just remove this sentence: waiting for the sales team to confirm.

**Pull request merged**